.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Request a new format specification to be created'
+labels: spec-request
+---
+<!--
+A request for a format specification consist of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete the fields without data and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. Must be a valid KSY. Meant to be ready to be copied as it is.
+meta:
+  id: # insert an id here. KS currently has no namespaces, so the id MUST be unique within all the repo and this property should be futureproof.
+  title: # insert format human-readable name here
+  application: # Insert the application human-readable name here
+  file-extension:
+    # a YAML list of file extensions here
+  encoding: UTF-8
+  endian: le
+  xref: # check if the following sources contain info about the format you request
+    mime: # MIME type
+    rfc: # Link to RFC
+    forensicswiki: # Page name on https://forensicswiki.org/
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/ 
+    wikidata: Q # ID on WikiData
+    fileformat: # Link to https://www.fileformat.info
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/
+    din:
+      id: # DIN standard identifier
```

Comment:
I would replace this entire block with a link to http://doc.kaitai.io/user_guide.html#meta-xref. That page already has an example `meta/xref` block and good explanations for all the keys. I don't think we can condense the meaning and expected format of each key into a short single-line comment, especially in a way that is understandable for newcomers.

Reply:
It is a template, IMHO it is easier to remove the unneeded fields rather than go to that page and copy the structure. Though I will add the link, thank you.